tests/containers/bci_repo.pm

```diff
@@ -62,12 +77,13 @@ sub run {
         container_exec($container, "zypper ar http://openqa.suse.de/assets/repo/$bci_repo BCI_TEST");
         container_exec($container, 'zypper ref', timeout => 180);
         container_exec($container, 'zypper lr -d');
-        record_info("Patterns", join(', ', @{$patterns->[$i]}));
+        record_info("Patterns", join(', ', @{$tdata->[$i]->{patterns}}));
         container_exec($container, 'zypper search -t pattern', timeout => 180);
-        container_exec($container, "zypper -n in -t pattern @{$patterns->[$i]}", timeout => 600);
+        container_exec($container, "zypper -n in -t pattern @{$tdata->[$i]->{patterns}}", timeout => 600);
         container_exec($container, "zypper -q -s 11 pa --orphaned | grep -v sles-release| tee -a repo.test", timeout => 600);
         container_exec($container, 'sed -i "s/ \+/ /g" repo.{org,test}', timeout => 600);
         container_exec($container, "diff repo.org repo.test", timeout => 600);
+        container_exec($container, "rpm -q @{$tdata->[$i]->{packages}}", timeout => 600);
```

Comment:
is it possible to print the output of this with `record_info`?

Reply:
Yes, but I would need to refactor `container_exec`. I do not really see the added value atm, but ofc it is possible.

Reply:
ah ok, right, let's ditch it, it was just to highlight the output.